drivers/wolfvision/eye14.cr

```diff
@@ -0,0 +1,215 @@
+require "digest/md5"
+require "placeos-driver/interface/muteable"
+require "placeos-driver/interface/powerable"
+require "placeos-driver/interface/switchable"
+require "placeos-driver/interface/camera"
+
+# require "tokenizer"
+
+module Wolfvision; end
+
+# Documentation: https://www.wolfvision.com/wolf/protocol_command_wolfvision/protocol/commands_eye-14.pdf
+# Ruby version: https://github.com/acaprojects/ruby-engine-drivers/tree/beta/modules/wolfvision
+
+class Wolfvision::Eye14 < PlaceOS::Driver
+  include PlaceOS::Driver::Interface::Powerable
+  include PlaceOS::Driver::Utilities::Transcoder
+  # include PlaceOS::Driver::Interface::Camera
+
+  # include PlaceOS::Driver::Interface::InputSelection(Power)
+  @channel : Channel(String) = Channel(String).new
+  @stable_power : Bool = true
+
+  @zoom_range = 0..3923
+  @iris_range = 0..4094
+
+  tcp_port 50915 # Need to go through an RS232 gatway
+  descriptive_name "WolfVision EYE-14"
+  generic_name :Camera
+
+  COMMANDS = {
+    power_on:        "\\x01\\x30\\x01\\x01",
+    power_off:       "\\x01\\x30\\x01\\x00",
```

Comment:
these commands should be binary, i.e. "\x01\x30\x01\x00"
however that might cause issues in crystal as all strings are UTF8 whereas the ruby code is ASCII (no escape characters) - Ruby also uses binary strings for most IO, whereas crystal has `Bytes` which might be more appropriate in this case

Reply:
binary commands implemented as suggested. Conversion to `Bytes` is occurring as expected.